src/components/Board/Board.container.js

```diff
@@ -1458,18 +1459,28 @@ const mapStateToProps = ({
   communicator,
   speech,
   scanner,
-  app: { displaySettings, navigationSettings, userData },
+  app: { displaySettings, navigationSettings, userData, isConnected },
   language: { lang }
 }) => {
   const activeCommunicatorId = communicator.activeCommunicatorId;
   const currentCommunicator = communicator.communicators.find(
     communicator => communicator.id === activeCommunicatorId
   );
   const activeBoardId = board.activeBoardId;
+  const currentVoice = speech.voices.find(
+    v => v.voiceURI === speech.options.voiceURI
+  );
   const emptyVoiceAlert =
     speech.voices.length > 0 && speech.options.voiceURI !== EMPTY_VOICES
       ? false
       : true;
+  const offlineVoiceAlert =
+    !isConnected &&
+    speech.voices.length &&
+    currentVoice &&
+    currentVoice.voiceSource === 'cloud'
```

Comment:
I think that we can use `currentVoice?.voiceSource === 'cloud'` to reduce de `&&` concatenation